cmd/backup-manager/app/util/remote.go

```diff
@@ -175,77 +201,74 @@ func newGcsStorage(qs *gcsQuery) (*blob.Bucket, error) {
 	}
 
 	// Create a *blob.Bucket.
-	bucket, err := gcsblob.OpenBucket(ctx, client, qs.bucket, nil)
+	bucket, err := gcsblob.OpenBucket(ctx, client, conf.bucket, nil)
 	if err != nil {
 		return nil, err
 	}
-	return blob.PrefixedBucket(bucket, qs.prefix), nil
+	return blob.PrefixedBucket(bucket, conf.prefix+"/"), nil
```

Comment:
What if `S3.Prefix=="/"`?

Reply:
we should trim the prefix before add a slash